spec/classes/init_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+describe 'Sentry' do
+  let(:facts) do
```

Comment:
This was to allow me to run the spec tests on my laptop, which is not running RHEL7.  Is there a better way to support running spec tests on platforms other than the target platform?